### DISCUSSION TITLE
throws Unhandled Error from clearFullRefreshStorage function

### DISCUSSION THREAD

**Discussion by veera-selvam-s** [vercel/next.js#43626] [Help]
I'm working on an app that should run when webStorage (local / session) is disabled. when i refresh the the tab it shows

**Unhandled Runtime Error**

TypeError: sessionStorage is null

this is from clearFullRefreshStorage

node_modules/next/dist/client/dev/error-overlay/hot-dev-client.js (274:8)

function clearFullRefreshStorage() {
    if (sessionStorage.getItem(FULL_REFRESH_STORAGE_KEY) !== 'ignore') sessionStorage.removeItem(FULL_REFRESH_STORAGE_KEY);
} 

this error is from above mentioned file. how can i handle this?

**Comment by icyJoseph** [comment]
What next version are you using?

I think in June 2022, through 2ff660c25a8cb8ceacfdefe858f25d8a04189d88, this was removed.

**Reply by veera-selvam-s** [reply]
this is my result for next info

    Operating System:
      Platform: darwin
      Arch: x64
      Version: Darwin Kernel Version 21.6.0: Thu Sep 29 20:12:57 PDT 2022; root:xnu-8020.240.7~1/RELEASE_X86_64
    Binaries:
      Node: 16.15.1
      npm: 8.11.0
      Yarn: 1.22.19
      pnpm: N/A
    Relevant packages:
      next: 12.1.0
      react: 17.0.2
      react-dom: 17.0.2

**Reply by icyJoseph** [reply]
You need to upgrade to next@12.2.0 at least, https://github.com/vercel/next.js/releases/tag/v12.2.0, introduced https://github.com/vercel/next.js/pull/37874

**Reply by veera-selvam-s** [reply]
thank you for quick reply

**Reply by icyJoseph** [reply]
Awesome, don't forget to mark as answered if it fixed your issue.
